src/shared/locales/all.js

```diff
@@ -16,6 +16,7 @@ import eleLocaleVi from 'element-ui/lib/locale/lang/vi'
 import eleLocaleZhCN from 'element-ui/lib/locale/lang/zh-CN'
 import eleLocaleZhTW from 'element-ui/lib/locale/lang/zh-TW'
 import eleLocaleUk from 'element-ui/lib/locale/lang/ua'
+import eleLocaleHu from 'element-ui/lib/locale/lang/hu'
```

Comment:
Please let import also be in alphabetical order.

Reply:
fixed